municipalities/beek-design-tokens/src/component/utrecht/table.tokens.json

```diff
@@ -0,0 +1,60 @@
+{

```

Comment:
The table should look more like the table on https://www.gemeentebeek.nl/gemeente-beek-maakt-subsidiebeleid-voor-verenigingen-stuk-eenvoudiger

Only think we will do differently is make the header look like the first column
<img width="418" height="320" alt="image" src="https://github.com/user-attachments/assets/4612e445-f751-4fd2-b327-8fb6174c7248" />